tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrationUtil.java

```diff
@@ -171,6 +171,34 @@ public static TaskProblem getTaskProblemFromSelection( ISelection selection )
         return problems;
     }
 
+    public static List<TaskProblem> getResolvedTaskProblemsFromResource( IResource resource )
+    {
+        final List<TaskProblem> problems = new ArrayList<>();
+
+        try
+        {
+            final IMarker[] markers =
+                resource.findMarkers( MigrationConstants.MARKER_TYPE, true, IResource.DEPTH_ZERO );
+
+            for( IMarker marker : markers )
+            {
+                TaskProblem taskProblem = markerToTaskProblem( marker );
+
+                boolean resolved = marker.getAttribute( "migrationProblem.resolved", false );
```

Comment:
Here you can check the taskPRoblem.isResolved() instead of checking the attribute directly.